### DISCUSSION TITLE
Pyenv is correctly configured but it doesn't start the desired python version

### DISCUSSION THREAD

**Discussion by epris** [pyenv/pyenv#2177] [Q&A]
Dear all,

I have installed pyenv by following the instructions reported on the main page.
If I execute
`pyenv versions`
I obtain the following result
```
system
* 3.7.7 (set by PYENV_VERSION environment variable)
```
But If i execute
`$ python --version`
the response is 
`Python 2.7.18`

I have tried to execute both pyenv global 3.7.7 and pyenv local 3.7.7 but If I write python3, the 2.7.18 version is started.
I'm using Ubuntu.
The PATH variable is set as shown in your tutorial.

What is wrong?

Best

**Comment by native-api** [comment]
Most probably, you didn't restart your login session after installing thus have no `shims` in your `PATH`.
In https://github.com/pyenv/pyenv/issues/2085, lifting the restart requirement is being discussed.
